resources/dicts/relationship_events/normal_interactions/comfortable/increase.json

```diff
@@ -239,5 +256,27 @@
 			"comfortable": "increase",
 			"trust": "increase"
 		}
+	},
+	{
+		"id": "comfort_inc_storm",
+		"interactions": [
+			"m_c comforted r_c through a particularly rough thunderstorm."
+		],
+		"reaction_random_cat": {
```

Comment:
might be good to stick some dislike decreases in here

Reply:
Fixed